cmd/antrea-agent/agent.go

```diff
@@ -64,17 +64,18 @@ const informerDefaultResync = 12 * time.Hour
 // run starts Antrea agent with the given options and waits for termination signal.
 func run(o *Options) error {
 	klog.Infof("Starting Antrea agent (version %s)", version.GetFullVersion())
+
+	// Setup signal capture: the first SIGTERM / SIGINT signal is handled gracefully and will
```

Comment:
I think we should use "Set up" which is a verb.

Reply:
Thanks. Updated.